bosh_cli/lib/cli/commands/job_management.rb

```diff
@@ -19,7 +19,7 @@ def start_job(job = '*', index_or_id = nil)
       usage 'stop'
       desc 'Stop all jobs/job/instance'
       option '--soft', 'Stop process only'
-      option '--hard', 'Power off VM'
+      option '--hard', 'Deletes the VM'
       option '--force', FORCE
```

Comment:
May want to use imperative form (Delete without s at the end) to be consistent with the other messages.